From b7807d5e6f62e35a4d4edbd8e7462ea9dbeba3d2 Mon Sep 17 00:00:00 2001
From: pratik Bhalgat <pratik_bit@codeaurora.org>
Date: Mon, 7 Jul 2014 20:03:33 +0530
Subject: [PATCH 2/4] wlan:Kbuild changes to compile in TDLS and out Channel
 Avoidance feature

Channel Avoidance feature depends on platform driver which do not exist.
Compilation flag for TDLS is not considered properly.
Addressing both issue with this commit.

Change-Id: Ifdb02e2a1376c44a341b8b20d16398dacf4af3a8
---
 Kbuild | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Kbuild b/Kbuild
index d799f24..a82af24 100644
--- a/Kbuild
+++ b/Kbuild
@@ -7,6 +7,9 @@ else
 endif
 
 ifeq ($(KERNEL_BUILD),1)
+	#Flag to enable TDLS feature
+	CONFIG_QCOM_TDLS := y
+
 	# These are provided in Android-based builds
 	# Need to explicitly define for Kernel-based builds
 	MODNAME := wlan
@@ -653,7 +656,7 @@ ifeq ($(CONFIG_ENABLE_LINUX_REG), y)
 CDEFINES += -DCONFIG_ENABLE_LINUX_REG
 endif
 
-CDEFINES += -DFEATURE_WLAN_CH_AVOID
+#CDEFINES += -DFEATURE_WLAN_CH_AVOID
 
 # Fix build for GCC 4.7
 EXTRA_CFLAGS += -Wno-maybe-uninitialized -Wno-unused-function
-- 
1.8.2.1

